tools/eslint/node_modules/es6-set/CHANGES

```diff
@@ -24,7 +24,7 @@ v0.1.0  --  2014.04.29
 * Remove comparator support (as it was removed from spec)
 * Provide @@toStringTag symbol, ad @@iterator symbols on iterators
 * Update to use latest dependencies versions
-* Improve interals
+* Improve internals
```

Comment:
This is an external dependency so we shouldn't fix it here, but rather submit a PR upstream to [the `es6-set` project](https://github.com/medikoo/es6-set).